Modules/Loadable/Sequences/Widgets/qMRMLSequenceBrowserPlayWidget.cxx

```diff
@@ -147,11 +149,16 @@ void qMRMLSequenceBrowserPlayWidget::updateWidgetFromMRML()
   bool recordingAllowed = d->SequenceBrowserNode->IsAnySequenceNodeRecording();
   bool playbackActive = d->SequenceBrowserNode->GetPlaybackActive();
   bool recordingActive = d->SequenceBrowserNode->GetRecordingActive();
-  d->pushButton_VcrRecord->setVisible(recordingAllowed);
+
   d->pushButton_VcrRecord->setEnabled(!playbackActive);
-  d->pushButton_Snapshot->setVisible(recordingAllowed);
   d->pushButton_Snapshot->setEnabled(!playbackActive && !recordingActive);
 
+  if (d->recordingControlsVisible)
```

Comment:
What would hide the record and snapshot buttons?
Instead of using a branch, simply replace 

    d->pushButton_Snapshot->setVisible(recordingAllowed);

by

    d->pushButton_Snapshot->setVisible(recordingAllowed && this->RecordingControlsVisible);

Reply:
It turns out that it should have been

```diff
-d->pushButton_Snapshot->setVisible(recordingAllowed && this->RecordingControlsVisible);
+d->pushButton_Snapshot->setVisible(recordingAllowed && d->RecordingControlsVisible);
```

See  #5896